app/presenters/content_item/parts.rb

```diff
@@ -87,7 +87,7 @@ def part_links
       parts.map.with_index(1) do |part, position|
         if part["slug"] != current_part["slug"]
           link_to part["title"], part["full_path"], class: "govuk-link",
-            data: {
+                                                    data: {
```

Comment:
seems undesirable

Reply:
Indeed. Same comment as [here](https://github.com/alphagov/finder-frontend/pull/2056#discussion_r418460305), really. 

Also worth noting the previous layout was a bit pants as well.

```
data: {
track_category: ... # ever heard of indentation?
```

Really the implementer could have just put each keyword/argument on it's own line.